leadtutes.tex

```diff
@@ -19,7 +19,9 @@ \subsubsection{Science Quality and Reliability Engineering \label{sect:square}}
 
 \item \gls{DM} documentation support, to include defining standards and providing tooling for documentation as well as some document writing
 
-\item Support of publicly released software products, including porting and distributing them according to the scientific community's needs
+\item Support of publicly software products build infrastructure e.g. Jenkins, and release tools e.g. container creation.
```

Comment:
yes it would seem i deleted the wrong word ..  in fact in mine it reads  "Support of software products build infrastructure" but apparenlty i had not pushed it ..

Reply:
yes and publicly